gradle/dependencies.gradle

```diff
@@ -57,6 +63,8 @@ dependencies {
     compileOnly fg.deobf("mezz.jei:jei-${project.jei_mcversion}-forge-api:${project.jei_version}")
     runtimeOnly fg.deobf("mezz.jei:jei-${project.jei_mcversion}-forge:${project.jei_version}")
 
+    implementation fg.deobf("curse.maven:jade-324717:${project.jade_id}")
```

Comment:
No, our CI system has problems with curse dependencies since it can not cache them
Find an official maven.

Reply:
Sadly, cursemaven **is** the official maven repo.  [Reference](https://jademc.readthedocs.io/en/latest/plugins19/getting-started/)

Reply:
Could we perhaps just cache it manually in the LDT maven?  What files would you need for that?

Reply:
No we can't we tried that with dynamic trees and it failed horribly.

Reply:
Why?  It's not like maven dir structures are all that complicated.  It's trivial to get a copy of the release jar and corresponding pom.  It's a bit harder if it needs the supplemental jars as well, but even that isn't all that hard, since only source-compatibility is needed and not binary-compatibility.

Reply:
As already said we tried this with dynmap that failed horribly. Then we tried to setup our own ci for their stuff that failed too because version numbers did not line up.


The onyk way this works properly is if they get a normal proper maven with proper coordinates and proper Metadata. This has been discussed to death and this is our official stance.

Reply:
This is not the official curse maven btw. The official one seems to use the proper name.

Reply:
The official one is probably fine. I would need to see it. However we just had problems with this one.

Reply:
iwfm (new commit)